.github/workflows/node.js.yml

```diff
@@ -13,15 +13,45 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14.x
+      - name: Use Node.js 18.x
         uses: actions/setup-node@v2
         with:
-          node-version: 14.x
+          node-version: 18.x
           cache: 'yarn'
       - run: yarn install
-      - name: Install dependencies in react-components
-        run: yarn install
+
+      # - run: yarn workspace @department-of-veterans-affairs/react-components run build
+      # - run: yarn workspace @department-of-veterans-affairs/web-components run build
+      # run tests individually because
+      # - run: yarn workspaces foreach --all -v run test
+      # is not working after upgrades related to storybook 7, eg node and yarn
+
+      - name: Install deps in packages
+        run: yarn workspaces foreach --all -v install
+
+      # - name: Test packages
+      #   run: yarn workspaces foreach --all -v run test
+
+      # - name: Install dependencies in react-components
+      #   run: yarn install
+      #   working-directory: packages/react-components
```

Comment:
Do we still need or want these steps combined? If so, can any of this be cleaned up?

Reply:
@jamigibbs I cleaned this action up (the `foreach` was still failing on install and test so I broke it up into three steps)